lib/regex.ts

```diff
@@ -1,6 +1 @@
-// @ts-nocheck
-const STRING_WITH_NO_SPACES_REGEX = /^\S*$/;
-
-module.exports = {
-  STRING_WITH_NO_SPACES_REGEX,
-};
+export const STRING_WITH_NO_SPACES_REGEX = /^\S*$/;
```

Comment:
Thought about maybe removing this file too, but figured regexes are confusing and this makes things clearer even though it's only used in two places

Reply:
I actually just killed this file in my PR here: https://github.com/HubSpot/hubspot-cli/pull/1222 . So keeping it around in your Pr is fine